package.json

```diff
@@ -25,7 +25,7 @@
     "node-libs-browser": "^2.2.1",
     "schema-utils": "^1.0.0",
     "tapable": "^1.1.3",
-    "terser-webpack-plugin": "^1.4.3",
+    "terser-webpack-plugin": "^4.2.3",
```

Comment:
CI is failing. Probably because `terser-webpack-plugin` [dropped support for Node <10.13](https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/CHANGELOG.md#300-2020-05-01), can you install the older version on CI when the node is if that is the case?

Reply:
Other than the CI failures, technically webpack4 still declares a Node 6+ requirement, so the requirement may need to be updated if it's not made backward compatible - although probably a nitpick given how ancient and EOL even Node 10 is now (security EOL 30 April 2021).

https://github.com/webpack/webpack/blob/dfffd6a241bf1d593b3fd31b4b279f96f4a4aab1/package.json#L95-L97